.github/workflows/build-ampd-release.yaml

```
@@ -0,0 +1,30 @@
+name: Amplifier - Build Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Github tag to release binaries for (reusing an existing tag will make the pipeline fail)
+        required: true
+        default: latest
+
+jobs:
+  release-binaries:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

for release builds, we should fix the OS version
```suggestion
        os: [ubuntu-22.04, macos-12]
```